installer/dev/install.cpp

```diff
@@ -10,51 +10,82 @@ using namespace Windows::System;
 
 namespace ProjectReunionInstaller {
 
-    winrt::hresult AddPackage(const Uri& packageUri)
+    HRESULT RegisterPackage(const std::wstring& packageFullName)
+    {
+        PackageManager packageManager;
+        const auto deploymentOperation{ packageManager.RegisterPackageByFullNameAsync(packageFullName, nullptr, DeploymentOptions::None) };
+        deploymentOperation.get();
+        if (deploymentOperation.Status() != AsyncStatus::Completed)
+        {
+            return static_cast<HRESULT>(deploymentOperation.ErrorCode());
```

Comment:
Register isn't expected to fail so `return LOG_IF_FAILED(static_cast<HRESULT>(...));` would be useful

Reply:
In my manual testing I recall it did fail, and the documentation does not indicate that it is not expected to fail, though there is no sample for register.